app/blueprints/assessments/templates/assessor_tasklist.html

```diff
@@ -147,41 +150,69 @@
             {% endif %}
         {% endfor %}
 
-        {% if sub_criteria_status_completed and g.access_controller.has_any_assessor_role and (not is_qa_complete) and (not flag_status) and not is_expression_of_interest %}
-            {{ assessment_complete(state, form.csrf_token, application_id)}}
+    {% if sub_criteria_status_completed and g.access_controller.has_any_assessor_role and (not is_qa_complete) and (not flag_status) and not is_expression_of_interest %}
+        {{ assessment_complete(state, form.csrf_token, application_id)}}
+    {% endif %}
+</div>
+</div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-third">
+        {{ comment(comments) }}
+        {% if display_comment_box != True and not display_comment_edit_box %}
+            <a
+                id="comment"
+                class="govuk-button secondary-button govuk-!-margin-top-2 govuk-!-margin-bottom-6"
+                type="submit"
+                data-module="govuk-button"
+                href="{{ url_for(
+                    "assessment_bp.application",
+                    application_id=application_id,
+                    add_comment="1",
+                    _anchor="comment"
+                ) }}">
+                {% if comments == None %} Add a comment
+                {% else %} Add another comment
+                {% endif %}
+        {# djlint:off #}</a>{# djlint:on #}
+        {% endif %}
+
+        {% if display_comment_box == True %}
```

Comment:
```suggestion
        {% if display_comment_box %}
```

nitpick, feel free to ignore

Reply:
fixed